L1Trigger/TrackTrigger/interface/TTStubAlgorithm_official.h

```diff
@@ -103,26 +108,39 @@ class ES_TTStubAlgorithm_official : public edm::ESProducer
     /// Windows
     std::vector< double >                setBarrelCut;
     std::vector< std::vector< double > > setRingCut;
+    std::vector< std::vector< double > > setTiltedCut;
 
     /// Z-matching
     bool  mPerformZMatchingPS;
     bool  mPerformZMatching2S;
 
+    bool  m_tilted;
+
   public:
     /// Constructor
     ES_TTStubAlgorithm_official( const edm::ParameterSet & p )
     {
       mPerformZMatchingPS =  p.getParameter< bool >("zMatchingPS");
       mPerformZMatching2S =  p.getParameter< bool >("zMatching2S");
+      m_tilted            =  p.getParameter< bool >("isTilted");
 
       setBarrelCut = p.getParameter< std::vector< double > >("BarrelCut");
 
-      std::vector< edm::ParameterSet > vPSet = p.getParameter< std::vector< edm::ParameterSet > >("EndcapCutSet");
+      std::vector< edm::ParameterSet > vPSet  = p.getParameter< std::vector< edm::ParameterSet > >("EndcapCutSet");
+      std::vector< edm::ParameterSet > vPSet2 = p.getParameter< std::vector< edm::ParameterSet > >("TiltedBarrelCutSet");
```

Comment:
If you know that this is going to store element for the tilted, you might want to change the name..